src/hott/10-trivial-fibrations.rzk.md

```diff
@@ -84,39 +93,49 @@ The following type asserts that the fibers of a type family are contractible.
 
 ## Projection equivalences
 
+From a projection equivalence, it's not hard to inhabit fibers:
+
 ```rzk
--- From a projection equivalence, it's not hard to inhabit fibers
 #def inhabited-fibers-is-equiv-projection
-  (A : U)
-  (B : A → U)
-  (ABprojequiv : is-equiv (Σ (x : A) , B x) A (total-space-projection A B))
-  (a : A)
+  ( A : U)
+  ( B : A → U)
+  ( ABprojequiv : is-equiv (Σ (x : A) , B x) A (total-space-projection A B))
+  ( a : A)
   : B a
   :=
     transport A B (first ((first (second ABprojequiv)) a)) a
       ( (second (second ABprojequiv)) a)
       ( second ((first (second ABprojequiv)) a))
+```
 
--- This is great but we need more coherence to show that the inhabited fibers
--- are contractible; the following proof fails
--- #def is-equiv-projection-implies-contractible-fibers
---    (A : U)
---    (B : A → U)
---    (ABprojequiv : is-equiv (Σ (x : A) , B x) A (total-space-projection A B))
---    : contractible-fibers A B
---    :=
---      \ x → (second ((first (first ABprojequiv)) x) ,
---      \ u → second-path-Σ A B ((first (first ABprojequiv)) x) (x , u)
---             ( (second (first ABprojequiv)) (x , u)) )
+This is great but we need more coherence to show that the inhabited fibers are
+contractible; the following proof fails:
 
+```text
+#def is-equiv-projection-implies-contractible-fibers
```

Comment:
I guess we need `#fail` command in Rzk to mark definitions that should fail (and check that it fails indeed).

Reply:
Issue created: https://github.com/rzk-lang/rzk/issues/75